### DISCUSSION TITLE
make all error; CrdGenerator error

### DISCUSSION THREAD

**Discussion by Chandranana** [strimzi/strimzi-kafka-operator#8280] [Q&A]
Team, i am getting error in running make all command for the build. Wanted some help in resolving the same :
The error is seen for a number of classes, below log is for one of the classes.
```
CrdGenerator: error: class io.strimzi.api.kafka.model.connect.build.Build is not annotated with @Buildable (io.strimzi.api.kafka.model.connect.build.BuildBuilder does not exist)
CrdGenerator: error: class io.strimzi.api.kafka.model.connect.build.Build is not annotated with @Buildable (io.strimzi.api.kafka.model.connect.build.BuildFluent does not exist)
CrdGenerator: error: class io.strimzi.api.kafka.model.connect.build.Build is not annotated with @Buildable (io.strimzi.api.kafka.model.connect.build.BuildFluentImpl does not exist) 
```
[ERROR] Failed to execute goal org.codehaus.mojo:exec-maven-plugin:1.6.0:exec (generate-crd-co-install-v1) on project api: Command execution failed.: Process exited with an error: 1 (Exit value: 1) -> [Help 1]
If anyone has any pointers on what could be the reason for the issue or if any known solutions please share.

**Comment by scholzj** [comment]
What exact command are you running and where? Did you do any changes? Is this a clean project?

**Reply by Chandranana** [reply]
> What exact command are you running and where? 
I am tryin to build strimzi-kafka-operator from source on power platform. Running the same commands as in strimzi-kafka-operator/.jenkins/Jenkinsfile-pr-s390x file. 

Now i am getting error while running the `make all` command executed from buildStrimziImages().


Did you do any changes?
No

Is this a clean project?
No, adding s390x support for Jenkins build.

**Reply by scholzj** [reply]
I have no idea how the Jenkins file works. So not sure if it has any specialties there.

With _clean project_ I meant if it is a freshly checkout out from Github or if you run `mvn clean` / `make clean` first as that might be important.

**Reply by Chandranana** [reply]
Ok thanks, please let me know if you have any clue like why it could report this error.
